scripts/hyperspectral/hyperspectral_workflow.sh

```diff
@@ -517,13 +518,13 @@ for ((fl_idx=0;fl_idx<${fl_nbr};fl_idx++)); do
 	typ_in_ENVI=$(grep '^data type' ${hdr_fl} | cut -d ' ' -f 4 | tr -d '\015')
 	xps_tm=$(grep 'current setting exposure' ${mtd_fl} | cut -d ':' -f 2 | tr -d '" ,\015' )
 	fl_clb="${drc_spt}/calibration_vnir_${xps_tm}ms.nc"
-	if [ "${wvl_nbr}" -eq 272 ]; then 
+	if [ "${wvl_nbr}" -eq 272 ] || [ "${wvl_nbr}" -eq 273 ]; then 
 	    flg_swir='Yes' # [flg] SWIR camera
 	elif [ "${wvl_nbr}" -eq 955 ]; then 
 	    flg_vnir='Yes' # [flg] VNIR camera
 	else
```

Comment:
> for posterity. but it means the initial workflow, which use to work with just raw+header files, will now require a third file (metadata) to complete.

No, I don't think we should do this for posterity. Mostly for readability and accuracy (e.g. wvl_nbr is a proxy for sensor type). With respect to querying wvl_nbr from metadata, I was thinking of other users wishing to reuse this code and generalize it so that it works with their hyperspectral sensors. They would be responsible for such generalizing the code.

Note that GitHub allows a reviewer either to comment or require changes. I am looking at a very small part of the code so I don't have sufficient context to request a nitpicky change like this. But I should have been more clear by saying "would it make more sense to ..." rather than "shouldn't this ...".